osmose_config.py

```diff
@@ -317,6 +317,7 @@ class gen(default_country):
 france_departement("bourgogne/saone_et_loire", 7397, "FR-71", include=[
     # Saône-et-Loire
     'merge_restaurant_FR_cg71',
+    'merge_pei_SDIS_71'
```

Comment:
Please can you rename the analyser file `analyser_merge_hydrants_FR.py`, and fix the name here.